modules/ROOT/pages/gov-cloud-standalone.adoc

```diff
@@ -0,0 +1,41 @@
+= Standalone Mule Support
```

Comment:
should the title specifically call out that this is for GovCloud?

"Standalone Mule Support in MuleSoft Government Cloud"

Reply:
Updated - thanks!